test/cpp/microbenchmarks/grpc_benchmark_config.bzl

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Configuration macros for grpc microbenchmarking"""
+
+def grpc_benchmark_args():
+    """Command line arguments for running a microbenchmark under bazel"""
```

Comment:
nit: "for running a microbenchmark under bazel" isn't accurate - "when run as a test, where we care about the pass/fail status, not about the performance measurements it yields" or sth like that is better.